root/views.py

```diff
@@ -54,6 +55,14 @@ def about(request):
 def sponsors(request):
     return render(request, 'sponsors.html', context={'page': 'sponsors'})
 
+
 def travelling(request):
     # Info taken from: https://www.mapsofindia.com/pune/travel-to-pune.html
     return render(request, 'travelling.html', context={'page': 'travelling'})
+
+
+def schedule(request):
+    with open(os.path.dirname(os.path.realpath(__file__)) + '/schedules.json', 'r') as fd:
+        data = json.load(fd)
+    context = data
```

Comment:
No need of declaring a separate variable named context I feel. You can directly pass data itself in the render function